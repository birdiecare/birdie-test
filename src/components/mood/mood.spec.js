import React from 'react';
import { shallow } from 'enzyme';

import Mood from './index';

const render = ({ mood = '' } = {}) => {
  const props = { mood };
  return shallow(<Mood {...props} />);
};

describe('Mood', () => {
  test('default render is as expected', () => expect(render()).toMatchSnapshot());

  test('sad is rendered is as expected', () => expect(render({ mood: 'sad' })).toMatchSnapshot());

  test('tired is rendered is as expected', () => expect(render({ mood: 'tired' })).toMatchSnapshot());

  test('confused is rendered is as expected', () => expect(render({ mood: 'confused' })).toMatchSnapshot());

  test('hungry is rendered is as expected', () => expect(render({ mood: 'hungry' })).toMatchSnapshot());
});
