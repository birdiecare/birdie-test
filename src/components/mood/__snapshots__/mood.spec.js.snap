// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mood confused is rendered is as expected 1`] = `
<TransferWithinAStationIcon
  color="action"
  style={
    Object {
      "fontSize": 200,
    }
  }
/>
`;

exports[`Mood default render is as expected 1`] = `
<SentimentVerySatisfiedIcon
  color="primary"
  style={
    Object {
      "fontSize": 200,
    }
  }
/>
`;

exports[`Mood hungry is rendered is as expected 1`] = `
<FastfoodIcon
  color="secondary"
  style={
    Object {
      "fontSize": 200,
    }
  }
/>
`;

exports[`Mood sad is rendered is as expected 1`] = `
<SentimentVeryDissatisfiedIcon
  color="error"
  style={
    Object {
      "fontSize": 200,
    }
  }
/>
`;

exports[`Mood tired is rendered is as expected 1`] = `
<HotelIcon
  color="primary"
  fontSize="large"
  style={
    Object {
      "fontSize": 200,
    }
  }
/>
`;
